pnpm-lock.yaml

```diff
@@ -10211,7 +10270,7 @@ snapshots:
 
   app-builder-bin@5.0.0-alpha.10: {}
 
-  app-builder-lib@25.1.8(dmg-builder@25.1.8)(electron-builder-squirrel-windows@25.1.8):
+  app-builder-lib@25.1.8(dmg-builder@25.1.8(electron-builder-squirrel-windows@25.1.8))(electron-builder-squirrel-windows@25.1.8(dmg-builder@25.1.8)):
```

Comment:
For some reason pnpm decided to also specify extra peer deps here. 🤷 Hopefully this behavior is gone once we update pnpm to v10.